photon-api/src/integTest/scala/com/linkedin/photon/ml/evaluation/ShardedAreaUnderROCCurveEvaluatorTest.scala

```diff
@@ -79,7 +79,7 @@ class ShardedAreaUnderROCCurveEvaluatorTest extends SparkTestUtils {
 
     val evaluator = getEvaluator(labels, ids)
     val actualResult = evaluator.evaluate(sc.parallelize(scores.map { case (id, score) =>
-      (id, ScoredGameDatum.getDefaultScoredDatumWithScore(score))
+      (id, ScoredGameDatum(score = score))
```

Comment:
Score isn't the first item in the constructor, so to create a new instance using a defined score but otherwise default values, we must declare which parameter we're providing